### ISSUE TITLE
DRGModdingAutomationScripts probably shouldn't be implemented as a submodule

### ISSUE THREAD

**Issue by samamstar** [DRG-Modding/FSD-Template#3]
When using a submodule; it's not just pulling the most recent version when you clone the template, it's also permanently linked to the DRGModdingAutomationScripts repo, making it impossible to commit changes to the GlobalConfig.ini

Options:
- Just copy the files and don't implement as submodule
- Alternative branch of DRGModdingAutomationScripts that uses a new config folder from outside itself, so that the user of the template can save their work

**Comment by Buckminsterfullerene02** [comment]
I've been meaning to address this for quite a while but I made a bit of a mess of things when I already did a U-turn on the DRGModdingAutomationScripts repo's existance. I think that having it as a submodule and asking end users to run the simple `git submodule --init` command is far too much to ask as well as that problem with committing changes to your templated repo. 

I will opt with simply doing yet another U-turn and copying files instead. Apologies for not coming back to this sooner. The next update will very likely be next major version.
